src/engine_constants/mod.rs

```diff
@@ -1951,3 +2058,120 @@ impl EngineConstants {
         Ok(())
     }
 }
+
+#[cfg(test)]
```

Comment:
most of those tests are unnecessary, they should only cover deserialization / eventual version upgrades (but right now we're adding the first major version of the format)